package.json

```diff
@@ -5,6 +5,11 @@
   "main": "src/index.ts",
   "author": "Agriculture Capstone Team",
   "license": "UNLICENSED",
+  "engines": {
```

Comment:
I think all of these should be using '>=' instead of '=' so that any version greater than specified should be ok

Reply:
I disagree, I think t o sync our development versions we should be using `=`. Any upgrade should be a conscious one.